### PR TITLE
GRD-422 - support list of files with labels and update to use pipenv only

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

**Comment by callunity** [comment]
Why remove the `release.sh` and `VERSION` files?

**Comment by callunity** [comment]
I tested this out in Modulator as I haven't yet installed 3.10.4 locally. Modulator bundles it up as:
```
$ sudo tree ~/modules/gsi/modulator/sw/Ubuntu18.04/vidarr-tools-0.3/
/home/harmstrong/modules/gsi/modulator/sw/Ubuntu18.04/vidarr-tools-0.3/
├── Pipfile
├── Pipfile.lock
├── README.md
├── scripts
│   └── vidarr-build
├── test
│   ├── (...lots)
└── vidarr
    ├── __init__.py
    └── wdl.py
```
Notably, it's not making `wdl2vidarr` or `miniwdl` available in `bin` , unlike how previous versions do - might any of our tools our processes depend on these being available in modulator?

(fwiw, the `scripts/vidarr-build` file is the contents of `vidarr-build.py`. And `wdl2vidarr` looks like it's `vidarr/wdl.py` so it's not like it's not available, just not where expected)

**Comment by mlaszloffy** [comment]
I think the build issue may be related to the tag/release version being removed - I pushed a fix to modulator, can you try again?

**Comment by callunity** [comment]
> I think the build issue may be related to the tag/release version being removed - I pushed a fix to modulator, can you try again?

Sorry, I should have said that I changed the modulator recipe to point to a commit because I knew the tag wouldn't exist yet. I rebuilt using the commit you specified and modulator still built using the same structure. I suspect it has to do with the change to `build_type: pipenv` as opposed to the previous modules, which used `build_type: pip_file`

**Review by callunity — APPROVED** [review]

File: Pipfile [hunk 1]
```diff
@@ -4,12 +4,12 @@ url = "https://pypi.org/simple"
...
+flake8 = "*"
+pytest = "*"
+pytest-datadir = "*"
 
 [packages]
-miniwdl = "==0.9.0"
-pytest = "==6.2.1"
-pytest-datadir = "==1.3.1"
-setuptools = "==51.0.0"
+miniwdl = "*"
```
[the remark this answers is not in the document]
Comment (mlaszloffy) [reply Pipfile:12]: Pipfile.lock is used by pipenv during install to fetch specific versions (and perform download checksums). Having pipfile version here as a wildcard will let the pipfile.lock be updated as needed.

File: scripts/vidarr-build.py [hunk 2]
```diff
@@ -203,8 +203,8 @@
     res = requests.post(registration_url, json=workflow)
     if res.status_code not in [200, 201, 409]:
         print(
-            f"Failed to register workflow version on {registration_url}: response status {res.status_code}}")
-        print(f"{res.json() if res.content}")  # only read the response body if it is not empty
+            f"Failed to register workflow version on {registration_url}: response status {res.status_code}")
+        print(f"{res.json() if res.content else ''}")  # only read the response body if it is not empty
```
[the remark this answers is not in the document]
Comment (mlaszloffy) [reply scripts/vidarr-build.py:207]: Yes I think that is cleaner, thanks! I'll fix this.

File: Pipfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (djcooke) [reply Pipfile:12]: 👍 I figure it's easier to control and keep track of in the pipfile, and then sub-dependencies are managed in the lock file, but this is probably just a preference

**Review by djcooke — APPROVED** [review]

**Review by asofmosq — APPROVED** [review]

**Pull request merged**